bubblewrap.c

```diff
@@ -408,17 +421,16 @@ acquire_caps (void)
       /* Drop root uid, but retain the required permitted caps */
       if (setuid (getuid ()) < 0)
         die_with_error ("unable to drop privs");
+      did_setuid = TRUE;
     }
 
   if (is_privileged)
     {
+      uint32_t caps0 = REQUIRED_SETUP_CAPS_0 | requested_caps[0];
```

Comment:
Isn't this a huge security hole if bubblewrap is installed setuid?  I think we should be verifying whether `getuid() == 0` or so.